doc/development/source/tutorial-settings.rst

```diff
@@ -130,6 +130,31 @@ Well-behaved widgets remember their settings - the state of their
 checkboxes and radio-buttons, the text in their line edits, the
 selections in their combo boxes and similar.
 
+Persisting defaults
+*******************
+Default setting values are not set in stone. As different users might
+prefer different defaults, Orange remembers which settings were used
+and stores them as the new defaults when the widget is deleted. Widgets
+removed from there on will start with the new defaults.
+
+Updated defaults are stored in user's profile based on the operation system.
+(%APPDATA%\Orange\<version>\widgets on windows,
+~/Library/Application\ Support/orange/<version>/widgets on macOS,
+~/.local/share/Orange/<version>/widgets on linux)
+Original default values can be restored by deleting files from this folder.
+This is done automatically when canvas is run with the
+--clear-widget-settings option.
```

Comment:
Original defaults can be restored by deleting this folder or by running Orange from command line with `--clear-widget-settings` option.